src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx

```diff
@@ -141,7 +141,12 @@ export function OrderLimitPriceField({ id }: OrderLimitPriceFieldProps) {
       }
     }
   }
-
+  console.log('toggle out', {
+    limitOrderKind: limitOrder.kind,
+    formattedLimitPrice,
+    baseTokenAmount,
+    quoteTokenAmount,
+  })
```

Comment:
If you want to debug, I'm totally ok with `debug`

Reply:
Agree!

Reply:
Forgot about this, thanks. I will remove